src/MercadoPago/Core/Block/Sales/Order/Totals/FinanceCost.php

```diff
@@ -33,7 +33,8 @@ public function getSource()
      */
     public function initTotals()
     {
-        if ((float)$this->getSource()->getFinanceCostAmount() == 0) {
+        if ((float)$this->getSource()->getFinanceCostAmount() == 0 
```

Comment:
Realizar o parse antes da comparação
if (((float)$this->getSource()->getFinanceCostAmount()) == 0